.github/workflows/roles-readme.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: create roles readme
+
+on:  # yamllint disable-line rule:truthy
```

Comment:
It would probably be good to also run it on pull-requests. This way the author can see if the argument-spec is correct. This should then have the same branches and path config as for pushes.